bucket/firefox-nightly.json

```diff
@@ -23,5 +25,27 @@
             "firefox.exe",
             "Firefox Nightly"
         ]
-    ]
+    ],
+    "checkver": {
+        "url": "https://archive.mozilla.org/pub/firefox/nightly/latest-mozilla-central/",
+        "re": "firefox-([\\d\\.]+a\\d+)\\.en-US"
+    },
+    "autoupdate": {
+        "architecture": {
+            "64bit": {
+                "url": "https://archive.mozilla.org/pub/firefox/nightly/latest-mozilla-central/firefox-$version.en-US.win64.zip",
+                "hash": {
+                    "find": "([a-f0-9]{64})\\ssha256\\s\\d+\\sfirefox-$version.en-US.win64.zip",
```

Comment:
```suggestion
                    "regex": "(^$sha256\\ssha256\\s\\d+\\s$basename",
```

Reply:
checkver -u can extract the SHA512 automatically. So I just removed this line